src/components/AddNote.js

```diff
@@ -4,72 +4,84 @@ import Grid from '@mui/material/Grid';
 import FormLabel from '@mui/material/FormLabel';
 import { CirclePicker } from 'react-color';
 import TextField from '@mui/material/TextField';
+import PropTypes from 'prop-types';
 
+const AddNote = ({handleAdd, handleColorChange, noteLabelColor, setNoteLabelColor}) => {
 
-const AddNote = ({handleAdd, handleColorChange, noteLabelColor,setNoteLabelColor}) => {
-    
-    const [count, setCount] = useState(300);
-    const [text, setText]= useState("");
-    const [noteColor, setNoteColor]= useState("#d4b055");
-    const [title, setTitle]= useState("");
+  const [ count, setCount ] = useState(300);
+  const [ text, setText ]= useState('');
+  const [ noteColor, setNoteColor ]= useState('#d4b055');
+  const [ title, setTitle ]= useState('');
 
-    const colorList = ['#d4b055','#ed8c77', '#c187d6', '#39d0db', '#e2fa84']
+  const colorList = [ '#d4b055', '#ed8c77', '#c187d6', '#39d0db', '#e2fa84' ];
 
-    const handleChangleLabelColor = (color) => {
-        if(color==="#d4b055"){
-            setNoteLabelColor("#f5cf6e")
-        }else if(color==="#ed8c77"){
-            setNoteLabelColor("#faa28e")
-        }else if(color==="#c187d6"){
-            setNoteLabelColor("#dfa2f5")
-        }else if(color==="#39d0db"){
-            setNoteLabelColor("#74eef7")
-        }else if(color==="#e2fa84"){
-            setNoteLabelColor("#e6f5ab")
-        }
+  const handleChangleLabelColor = (color) => {
```

Comment:
use switch statement instead of if, call setNoteLabelColor only once, and define object with hexa: color pairs